samples/update-intent.js

```diff
@@ -0,0 +1,59 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+process.env.DEBUG = 'dialogflow:debug'; // enables lib debugging statements
+
+async function main(projectId, intentId, agentId, location, displayName) {
+  // [START dialogflow_cx_update_intent]
+
+  //await updateIntent("valiant-marker-319718","8dcfc58a-7fc3-4469-ae60-77402e98c933","a02e8272-a62d-4bf5-b235-b365ce4c4137","global","testFieldMask");
```

Comment:
Usually this comment looks like `Import the library`. Seems a bit difficult to decipher what this comment is saying.